opal/views.py

```diff
@@ -71,6 +71,14 @@ def get_context_data(self, **kwargs):
         context['list_slug'] = list_slug
         context['patient_list'] = self.patient_list
         context['lists'] = PatientList.for_user(self.request.user)
+
+        context['list_group'] = None
+        if self.patient_list:
+            group = TabbedPatientListGroup.for_list(self.patient_list)
+            if group:
```

Comment:
Not entirely sure what you mean ? 

 `member_lists_for_user` gets the iterable of `PatientLists` that make up this group, `group.visible_to` determines whether the group itself is visible to the user in question, `for_list` figures out if this List is in a group.

The difference is useful because for instance I might want to restrict who can see the group UI - for instance I have three different team lists, and all doctors can, if the choose, navigate to all three lists, but will be mostly working from one. However the admin/nursing staff cover all three teams because they're on the same physical ward and so move in between all day. I can do this by overriding `visible_to` to take into account user roles etc